torch/cuda/amp/grad_scaler.py

```diff
@@ -677,3 +694,30 @@ def _check_inf_per_device(self, optimizer: torch.optim.Optimizer) -> Dict[str, A
 
     def _found_inf_per_device(self, optimizer: torch.optim.Optimizer) -> Dict[str, Any]:
         return self._per_optimizer_states[id(optimizer)]["found_inf_per_device"]
+
+    def _update_scale_(
+        self,
+        _scale: torch.Tensor,
+        _growth_tracker: torch.Tensor,
```

Comment:
```suggestion
        scale: torch.Tensor,
        growth_tracker: torch.Tensor,
```
I think this  is more conventional